Frontend/src/Components/Temperature/Temperature.js

```diff
@@ -0,0 +1,69 @@
+import {Flex, useColorMode} from "@chakra-ui/react";
+import RangeRow from "../Shared/RangeRow";
+import CONSTANTS from "../../data-constants.json";
+import getColor from "../Shared/colors";
+
+export default function Temperature(props) {
+    const { colorMode } = useColorMode();
+    const headerBg = getColor("header", colorMode);
+
+    return (
+        <Flex flex="auto" direction="column" overflowY="scroll">
+            <RangeRow
+                dataTitle="Motor"
+                dataValue={props.data?.motor_temp[0] ?? -1.0}
+                dataConstant={CONSTANTS.motor_temp}
+
+                bg={headerBg}
+            />
+            <RangeRow
+                dataTitle="Road"
+                dataValue={props.data?.road_temp[0] ?? -1.0}
+                dataConstant={CONSTANTS.road_temp}
+            />
+            <RangeRow
+                dataTitle="Brake"
+                dataValue={props.data?.brake_temp[0] ?? -1.0}
+                dataConstant={CONSTANTS.brake_temp}
+
+                bg={headerBg}
+            />
+            <RangeRow
+                dataTitle="Air"
+                dataValue={props.data?.air_temp[0] ?? -1.0}
+                dataConstant={CONSTANTS.air_temp}
+            />
+            <RangeRow
+                dataTitle="DCDC"
+                dataValue={props.data?.dcdc_temp[0] ?? -1.0}
+                dataConstant={CONSTANTS.dcdc_temp}
+
+                bg={headerBg}
+            />
+            <RangeRow
+                dataTitle="Motor Controller"
+                dataValue={props.data?.motor_controller_temp[0] ?? -1.0}
+                dataConstant={CONSTANTS.motor_controller_temp}
+            />
+            <RangeRow
+                dataTitle="Cabin"
+                dataValue={props.data?.cabin_temp[0] ?? -1.0}
+                dataConstant={CONSTANTS.cabin_temp}
+
+                bg={headerBg}
+            />
+            <RangeRow
+                dataTitle="Main IO"
+                dataValue={props.data?.mainIO_temp[0] ?? -1.0}
+                dataConstant={CONSTANTS.mainIO_temp}
+            />
+            <RangeRow
+                dataTitle="Driver IO temp"
```

Comment:
```suggestion
                dataTitle="Driver IO"
```
Staying consistent with the other row labels. Please make this change in your first commit rather than making an additional commit to fix it.